velox/experimental/wave/exec/tests/CMakeLists.txt

```diff
@@ -55,7 +55,8 @@ target_link_libraries(
   gflags::gflags
   glog::glog
   fmt::fmt
-  ${FILESYSTEM})
+  ${FILESYSTEM}
```

Comment:
`${FILESYSTEM}` is no longer needed as we have raised our minimum supported gcc version.